crates/pessimistic-proof-test-suite/src/forest.rs

```diff
@@ -100,10 +100,10 @@ impl Forest {
             },
         };
 
-        self.l1_info_tree.add_leaf(l1_leaf.hash());
+        self.l1_info_tree.add_leaf(l1_leaf.hash()).unwrap();
```

Comment:
Same here. No naked `unwrap`s. Either handle the Result::Err case here for use `?` to pass it up the call stack.

Reply:
> This function (and everything else in the pessimistic-proof-test-suite crate) is only used for testing, so I think it's fine to use unwrap here.

ditto